package/kedro_viz/data_access/repositories/runs.py

```diff
@@ -71,6 +76,16 @@ def get_user_run_details(self, run_id: str) -> Optional[UserRunDetailsModel]:
             .first()
         )
 
+    @check_db_session
+    def get_new_runs(self) -> Optional[Iterable[RunModel]]:
```

Comment:
I prefer to keep these interfaces separate in case we need to add more logic to each use case. It's an old habit, e.g. if we end up caching all runs somewhere we won't need to go to the DB to fetch them. We will almost always have to do it for new runs.